ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/CommandDeduplicationIT.scala

```diff
@@ -620,8 +614,13 @@ final class CommandDeduplicationIT(
           .getTimeModel()
           .flatMap(response => {
             ledger.delayMechanism.delayBy(
-              response.getTimeModel.getMaxSkew.asScala +
-                2 * response.getTimeModel.getMinSkew.asScala
+              Durations.scaleDuration(
+                Durations.asFiniteDuration(
+                  2 * (response.getTimeModel.getMaxSkew.asScala +
+                    response.getTimeModel.getMinSkew.asScala)
+                ),
+                timeoutScaleFactor,
```

Comment:
This should always have been here, i.e. this is a fix, right?

Reply:
Yes, it should always use the time scale